docs/include/plugin_header-nonstandard.asciidoc

```diff
@@ -0,0 +1,25 @@
+[subs="attributes"]
+++++
+<titleabbrev>{plugin}</titleabbrev>
+++++
+
+* Plugin version: {version}
+* Released on: {release_date}
+* {changelog_url}[Changelog]
+
+For other versions, see the
+{lsplugindocs}/{type}-{plugin}-index.html[Versioned plugin docs].
+
+ifeval::["{default_plugin}"=="0"]
+
+==== Installation
+
+For plugins not bundled by default, it is easy to install by running +bin/logstash-plugin install logstash-{type}-{plugin}+. See {logstash-ref}/working-with-plugins.html[Working with plugins] for more details.
+
+endif::[]
+
+==== Getting help
+
+For questions about the plugin, open a topic in the http://discuss.elastic.co[Discuss] forums. For bugs or feature requests, open an issue in https://github.com/elastic/logstash-{type}-{plugin}[Github].
```

Comment:
I had to create a net new header file, but the only difference from plugin_header.asciidoc is the path to the GitHub repo (`elastic` instead of `logstash-plugins`).